scripts/fetch

```diff
@@ -9,22 +9,22 @@ cores=$(nproc)
 manifest_dir="${MANIFEST_DIR?}"
 manifest_repo="${MANIFEST_REPO?}"
 
-mkdir -p "${base_dir}" "${manifest_dir}"
+mkdir --parents "${base_dir}" "${manifest_dir}"
 
-rsync -Pav "${manifest_repo}/" "${manifest_dir}/"
+rsync -P --archive --verbose "${manifest_repo}/" "${manifest_dir}/"
 
 [ -d "${manifest_dir}/.git" ] || {
 	export GIT_WORK_TREE="$manifest_dir"
 	export GIT_DIR="${manifest_dir}/.git"
 	git init
 	git add .
-	git commit -a -m "automated cache commit"
+	git commit --all --message "automated cache commit"
 }
 
 cd "${base_dir}"
 
-repo init -u "${manifest_dir}" -m base.xml
+repo init --manifest-url "${manifest_dir}" --manifest-name base.xml
```

Comment:
this is a good change. `repo` isn't as common of a tool and I don't find the short args memorable.

Reply:
@daurnimator We have a similar knowledge of the command flags as it seems. The above are known, short or long. The repo command are where I really needed to look up and which was the driving force to use the long variants.

Reply:
I'd be happy to merge these `repo` changes alone for now and debate over the others later?

Reply:
Sure, I guess there is no need to go crazy about those changes. Feel free to merge #29. You might as well close this PR (#27) and I will be more conservative with similar changes in the future.